tests/administrators/test_tasks.py

```diff
@@ -0,0 +1,55 @@
+from sqlalchemy.ext.asyncio import AsyncEngine, AsyncSession
+
+from virtool.administrators.tasks import PromoteAdministratorsTask
+from virtool.pg.utils import get_row_by_id
+from virtool.tasks.models import Task
+from virtool.utils import get_temp_dir
+
+
+async def test_promote_administrators(
+    data_layer,
+    mongo,
+    pg: AsyncEngine,
+    snapshot,
+    static_time,
+    tmpdir,
+    fake2,
+):
+
+    await fake2.users.create()
+
+    user = await fake2.users.create()
+
+    async with mongo.create_session() as session:
```

Comment:
Don't need a session for this.

They are slower and data consistency doesn't matter so much for tests.